saleor/graphql/product/tests/test_product.py

```diff
@@ -8684,32 +8825,6 @@ def test_update_product_with_non_existing_attribute(
     ]
 
 
-def test_update_product_with_no_attribute_slug_or_id(
-    staff_api_client, product, permission_manage_products, color_attribute
-):
```

Comment:
Right now the `id` requirement is ensured by graphql.
There was no possibility to provide a `slug` for a longer time.